tools/pactlinfo.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+# Licensed under GPLv3.
```

Comment:
where does this come from? We should not blindly add stuff here, there's also no copyright information?

Reply:
The script is copied and modified from the file _audio_data.py in the package apport-symptoms. The license is from that file. Is the GPLv2 OK for us?

Reply:
>  Is the GPLv3 OK for us?

No, not in this case.

Even if it were OK you should never ever remove a copyright.

GPL aside, code re-use does not mean "copy/paste", you should try to import it first.

When really impossible re-use as is and copy cannot be avoided you should first commit the initial version and then the modifications in a different commit for version tracking purposes.

Pretty much everything is wrong.

Reply:
If so, it seems we can't reuse the codes. The original file is much complicated. Imposing it directly will cause failure, I did a little change in the original code. "When really impossible re-use as is and copy cannot be avoided you should first commit the initial version and then the modifications in a different commit for version tracking purposes." is a good suggestion.

Anyway, as we are concerning about the GPL. I will try to redesign the code and won't user GPLv3.